### ISSUE TITLE
LoggerResource can not change Logger with classes out of the classpath

### ISSUE THREAD

**Issue by jdotcms** [dotCMS/core#19183]
Context:

Was trying to switch the level of a class in an OSGI bundle. The class does not exists on the dotCMS classpath so can not get the logger by class, must be by name since by name it is reachable.

**Comment by jdotcms** [comment]
[PR](https://github.com/dotCMS/core/pull/19185/files)

**Comment by erickgonzalez** [comment]
Working as it should.

Steps Tested:
* Generated the jar of the `visitoripconditionlet` OSGI plugin
* Uploaded the fragment and the jar
* Call the API to check the level of the logger `{{serverURL}}/api/v1/logger/com.dotmarketing.osgi.ruleengine.conditionlet.Activator`
* Change the level of the logger
```
POST 
{{serverURL}}/api/v1/logger/

 {
        "level": "DEBUG",
        "name": "com.dotmarketing.osgi.ruleengine.conditionlet.Activator"
   
}
```
* Check again the level of the logger

**Comment by bryanboza** [comment]
Fixed, tested on release-5.3.8 // Postgres // FF... According to the provided steps everything works as expected
